hosting_docs/source/reference/firefighting/general.rst

```diff
@@ -805,7 +805,12 @@ Check out :ref:`reference/firefighting/celery:Celery Firefighting Guide`.
 Monitoring
 ----------
 
-Sometimes it's helpful to check "Flower", a UI for celery monitoring http://hqcelery1.internal-va.commcarehq.org:5555/ (you have to be VPN'd in).
+Sometimes it's helpful to check "Flower", a UI for monitoring Celery.
+
+The URL for Flower in Dimagi's production environment is
+http://hqcelery1.internal-va.commcarehq.org:5555/ . To connect, run ::
```

Comment:
We could make this more general, by suggesting that this is the server configured in app-processes.yml to run flower process.